src/postprocessing/pp-h265.c

```diff
@@ -582,6 +582,8 @@ int janus_pp_h265_process(FILE *file, janus_pp_frame_packet *list, int *working)
 				/* Skip the FU header */
 				buffer += 3;
 				len -= 3;
+			} else {
+				JANUS_LOG(LOG_HUGE, "unhandled nalu type: %d\n", type);
```

Comment:
I don't think this else is needed here, there's not anything we do with it anyway.

Reply:
I'll merge and remove this myself. Thanks!